### DISCUSSION TITLE
Step 2 - Mortar Contact

### DISCUSSION THREAD

**Discussion by DYLDYLDYL** [idaholab/moose#19935] [Q&A Modules: Solid mechanics] [answered]
![2021-12-31 21-31-24屏幕截图](https://user-images.githubusercontent.com/91199452/147825862-55723374-996d-4bcc-9b95-b8c54e05bf81.png)
[step02.zip](https://github.com/idaholab/moose/files/7796303/step02.zip)
36/5000 
I encountered the following error while running tutorial step02.i. (No changes have been made to the input file)

**Comment by GiudGiud** [comment, marked as answer]
Hello

Which tutorial in particular are you talking about? The one in modules/contact/tutorials ?
As listed in the posting guidelines, please do not post screenshots, but copy paste the output from the terminal, formatted with the triple quotes.  https://github.com/idaholab/moose/discussions/18270

Guillaume

**Reply by DYLDYLDYL** [reply]
It's in modules/contact/tutorials/introductions/step02.i. 

*** ERROR ***
mesh: The 'mortar' formulation requires 'mesh' to be supplied

Thanks a lot

**Reply by DYLDYLDYL** [reply]
#18791

**Reply by GiudGiud** [reply]
This runs fine for me. Please update moose and see if the issue is still present. 
What is your current commit? (see output of `git log -1`)

**Reply by DYLDYLDYL** [reply]
Thank you for reply！
Merge: 02c3ce777a 58219425a3
Author: moosetest <bounces@inl.gov>
Date:   Wed Jul 28 13:21:50 2021 -0600
    Merge commit '58219425a3180a3429cc37d840a201fa3acf58f6'

I update by `conda update --all`,
I have no folder modules/contact/tutorials, 
I copy step02.i from website.

**Reply by GiudGiud** [reply]
This commit is pretty old, while the tutorial was added recently. 
To update moose, you need to use git. The conda command updates libmesh and other dependencies. 
See this page:
https://mooseframework.inl.gov/getting_started/installation/update_moose.html

**Reply by DYLDYLDYL** [reply]
Thanks！
